src/utils/ast.js

```diff
@@ -45,6 +45,6 @@ export function findEmptyLines(
     return [];
   }
   const lineCount = selectedSource.text.split("\n").length;
-  const sourceLines = range(1, lineCount);
+  const sourceLines = range(1, lineCount + 1);
```

Comment:
@jasonLaster This was the issue in the screenshot I sent you with the last line looking debuggable:
<img width="334" alt="screen shot 2018-03-19 at 4 37 22 pm" src="https://user-images.githubusercontent.com/132260/37741961-cee612c4-2d20-11e8-870b-9e825b760a9e.png">

Reply:
Awesome catch!